docs/cbc_sdk.platform.rst

```diff
@@ -52,6 +52,14 @@ cbc\_sdk.platform.jobs module
    :undoc-members:
    :show-inheritance:
 
+cbc\_sdk.platform.observations module
+----------------------------------
```

Comment:
the underline has to go to the end of the letters

Reply:
Fixed in the other PR, which will be merged into this one.